fhir-server/liberty-config/server.xml

```diff
@@ -151,6 +151,7 @@
         <fileset dir="${shared.resource.dir}/lib/derby" includes="*.jar"/>
         <fileset dir="${shared.resource.dir}/lib/db2" includes="*.jar"/>
         <fileset dir="${shared.resource.dir}/lib/fhir" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/postgresql" includes="*.jar"/>
```

Comment:
for consistency, move this entry one line up (before fhir) 
persistence libraries then fhir

Reply:
done